pkg/controller/memberstatus/memberstatus_controller_test.go

```diff
@@ -78,6 +100,7 @@ func TestOverallStatusCondition(t *testing.T) {
 		assert.Equal(t, requeueResult, res)
 		AssertThatMemberStatus(t, req.Namespace, requestName, fakeClient).
 			HasCondition(ComponentsNotReady(string(hostConnection)))
+		AssertThatMemberStatus(t, req.Namespace, requestName, fakeClient).HasHostConnectionConditionErrorMsg(errMsgHostConnectionNotFound)
```

Comment:
We usually do not use const from runtime in the test. It's better to explicitly copy the strings.
```suggestion
		AssertThatMemberStatus(t, req.Namespace, requestName, fakeClient).HasHostConnectionConditionErrorMsg("the host connection was not found")
```
The reason for that is if we accidentally change the const to something wrong ("blah-blah" for example). Then the test will still pass. Using strings allows as to test the constants too.

Reply:
Makes sense. Updated in c4d725ad046b4bf1e2f175f1bea36aa148d68ec3